mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoImplementationGenerator.java

```diff
@@ -86,6 +134,20 @@ protected String getFileName() {
             .addParameter(MapperContext.class, "context")
             .addStatement("this.context = context");
 
+    for (PartialClassGenerator methodGenerator : methodGenerators) {
+      methodGenerator.addMembers(classBuilder);
+      methodGenerator.addConstructorInstructions(constructorBuilder);
+    }
+
+    for (Map.Entry<ClassName, String> entry : entityHelperFields.entrySet()) {
+      ClassName fieldTypeName = entry.getKey();
+      String fieldName = entry.getValue();
+
+      classBuilder.addField(
+          FieldSpec.builder(fieldTypeName, fieldName, Modifier.PRIVATE, Modifier.FINAL).build());
+      constructorBuilder.addStatement("this.$L = new $T(context)", fieldName, fieldTypeName);
```

Comment:
What will be a type of a `context` argument of the generated constructor?

Reply:
`MapperContext`. It's in the `mapper-runtime` module.